html/browsers/windows/noreferrer.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<title>rel=noreferrer nullifies window.opener</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  async_test(function(t) {
+    localStorage.clear()
```

Comment:
How is this going to interact with other tests that exercise `localStorage`? @jgraham, doesn't the runner run things in parallel?

One option might be to run this test on a unique subdomain so that it has it's own storage partition.